docs/QUICKSTART.md

```diff
@@ -0,0 +1,70 @@
+# Install
+
+Super quick start, for knowledgeable individuals.
+
+- Use Go >= 1.11
+- Git clone as usual
+- `make install`
+- These should now work:
+  - `dcrlnd --version`
+  - `dcrlncli --version`
+
+# Configuration
+
+Create the config file `~/.dcrlnd/dcrlnd.conf`:
```

Comment:
Maybe the OS differences can be quote here.

Reply:
Added.